distribution/ddf-common/src/main/resources-filtered/etc/system.properties

```diff
@@ -59,6 +59,13 @@ org.codice.ddf.system.port=_DO_NOT_EXPAND_${org.codice.ddf.system.httpsPort}
 org.codice.ddf.system.internalHttpsPort=_DO_NOT_EXPAND_${org.codice.ddf.system.httpsPort}
```

Comment:
❓ These might be redundant now. Do we need to deconflict the two sets of properties now that `internal` is the default behavior and the full set of `external` properties can be configured externally? I think the comment on line 52 is inaccurate now.

Reply:
Essentially reduce it down to this? : 
``` org.codice.ddf.system.protocol=https://
 org.codice.ddf.system.hostname=localhost
 org.codice.ddf.system.httpsPort=8993
 org.codice.ddf.system.httpPort=8181
 org.codice.ddf.system.port=_DO_NOT_EXPAND_${org.codice.ddf.system.httpsPort}
 org.codice.ddf.system.rootContext=/services

#useful when DDF is behind a proxy and the URL a user sees is different than the local url DDF uses
org.codice.ddf.external.protocol=https://
org.codice.ddf.external.hostname=localhost
org.codice.ddf.external.httpsPort=8993
org.codice.ddf.external.httpPort=8181
org.codice.ddf.external.port=_DO_NOT_EXPAND_${org.codice.ddf.external.httpsPort}

```